velox/vector/arrow/Bridge.cpp

```diff
@@ -212,6 +213,28 @@ static void releaseArrowSchema(ArrowSchema* arrowSchema) {
   arrowSchema->private_data = nullptr;
 }
 
+const char* exportArrowFormatTimestampStr(
+    const ArrowOptions& options,
+    std::string& formatBuffer) {
+  switch (options.timestampUnit) {
+    case TimestampUnit::kSecond:
+      formatBuffer = fmt::format("tss:{}", options.timestampTimeZone);
+      break;
+    case TimestampUnit::kMilli:
+      formatBuffer = fmt::format("tsm:{}", options.timestampTimeZone);
+      break;
+    case TimestampUnit::kMicro:
+      formatBuffer = fmt::format("tsu:{}", options.timestampTimeZone);
+      break;
+    case TimestampUnit::kNano:
+      formatBuffer = fmt::format("tsn:{}", options.timestampTimeZone);
+      break;
+    default:
+      VELOX_UNREACHABLE();
+  }
+  return formatBuffer.c_str();
```

Comment:
```
Status Visit(const TimestampType& type) {
    switch (type.unit()) {
      case TimeUnit::SECOND:
        export_.format_ = "tss:";
        break;
      case TimeUnit::MILLI:
        export_.format_ = "tsm:";
        break;
      case TimeUnit::MICRO:
        export_.format_ = "tsu:";
        break;
      case TimeUnit::NANO:
        export_.format_ = "tsn:";
        break;
      default:
        return Status::Invalid("Invalid time unit for Timestamp: ", type.unit());
    }
    export_.format_ += type.timezone();
    return Status::OK();
  }
```
The arrow implementation is this one, I think it is better than fmt::format.
The compiler will optimize string +=

Reply:
@jinchengchenghh Updated.